GettingToKnowPython/HomeWorks/HomeWork_1/Task1(2).py

```diff
@@ -0,0 +1,15 @@
+"""Задача 2: Найдите сумму цифр трехзначного числа.
+*Пример:*
+123 -> 6 (1 + 2 + 3)
+100 -> 1 (1 + 0 + 0) |
+"""
+N = 257 #n - трехзначное число
+n = int(N)
+
+dig_1 = n % 10
+n = n // 10
+dig_2 = n % 10
+n = n // 10
```

Comment:
Можно обойтись без одной строчки.
```suggestion
dig_1 = n % 10
dig_2 = n // 10 % 10
n = n // 100
```